src/data-publication/reports/tables/Aggregate1.jsx

```diff
@@ -37,15 +40,58 @@ const renderDispositionValues = (values, key, key2) => {
   return values.map((value, index) => {
     return [
       <td key={`count-${key}-${key2}-${index}`}>{value.count}</td>,
-      <td key={`value-${key}-${key2}-${index}`}>{value.value}</td>
+      <td key={`value-${key}-${key2}-${index}`}>{value.value}</td>,
     ]
   })
 }
 
-const Aggregate1 = React.forwardRef((props, ref) => {
-  if (!props.report) return null
+/**
+ * Reducer function to aggregate Disposition data by dispositionName
+ * @param {Object} prev
+ * @param {Object} curr
+ * @returns Array[Object]
+ */
+const dispositionReducer = (prev, curr) => {
+  // Initialize tracker object for this dispositionName if necessary
+  if (!prev[curr.dispositionName])
+    prev[curr.dispositionName] = {
+      dispositionName: curr.dispositionName,
+      value: 0,
+      count: 0,
+    }
+
+  // Summarize content
+  prev[curr.dispositionName].count += curr.count
+  prev[curr.dispositionName].value += curr.value
+  return prev
+}
+
+/**
+ * Temporary Fix: Identifies unaggregated Disposition entries and aggregates them
+ * @param {Array} data
+ */
+const fixUnaggregatedDispositions = data => {
+  const idxsNeedAggregation = []
+
+  data.forEach(
+    (x, idx) =>
+      x.dispositions.some(dispo => dispo.values.length > 7) &&
+      idxsNeedAggregation.push(idx)
+  )
+
+  // Aggregate un-aggregated disposition data
+  idxsNeedAggregation.forEach(i => {
+    data[i].dispositions.forEach(d => {
+      const obj = d.values.reduce(dispositionReducer, {})
+      d.values = Object.keys(obj).map(k => obj[k])
+    })
+  })
 
-  const sortedTracts = props.report.tracts.sort(function(tractA, tractB) {
+  return data
+}
+
+export const sortAndFix = (report) => {
```

Comment:
Should we move the `sortAndFix` function to a utility file? It is being used in both `Aggregate` table components.

Reply:
`sortAndFix` in each Aggregate file do slightly different things, since the `Fix` function is different for each.  But we can refactor the `sort` part, since that is the shared...or maybe change `sortAndFix` to take the `Fix` fn as a parameter. 
 
 Will take a look.

Reply:
Add a parameter to take a `fix` function.